bool rear_red_spot_led
bool rear_spot_leds
bool rear_left_yellow_led
bool rear_right_yellow_led
bool front_left_yellow_led
bool front_right_yellow_led
bool front_spot_leds
bool spare





